### PR TITLE
pulseeffects: update to 5.0.4 & fix greyed-out effects

### PR DESCRIPTION
**Pull request by Logarithmus**
- New package: lsp-plugins-1.1.30
- zita-convolver-devel: add fftw-devel dependency
- New package: zam-plugins-3.14
- pulseeffects: update to 5.0.4 & fix greyed-out effects
- pulseeffects-legacy: fix greyed out effects

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by paper42** [comment]
Also, "fix greyed out effects" is confusing to me. Effects which need packages that are not installed are greyed out. I don't think pulseeffects needs to depend on all its plugins. I am pretty sure you should make a PR for each package separately.

File: srcpkgs/pulseeffects-legacy/template [hunk 1]
```diff
@@ -4,18 +4,16 @@ version=4.8.5
 revision=1
 wrksrc="pulseeffects-${version}"
 build_style=meson
-hostmakedepends="itstool pkg-config gettext"
+hostmakedepends="itstool pkgconf gettext"
```
Comment (paper42) [inline srcpkgs/pulseeffects-legacy/template:7]: why?

**Comment by Logarithmus** [comment]
Because `pkgconf` is faster and better in any way: http://pkgconf.org/features.html

File: srcpkgs/pulseeffects-legacy/template (hunk 1 above)
Comment (Logarithmus) [reply srcpkgs/pulseeffects-legacy/template:7]: @paper42 because `pkgconf` is faster and better in any way: http://pkgconf.org/features.html

Comment (paper42) [reply srcpkgs/pulseeffects-legacy/template:7]: almost every template uses pkg-config, so I think you should stick to the convention

**Comment by Logarithmus** [comment]
> Also, "fix greyed out effects" is confusing to me. Effects which need packages that are not installed are greyed out. I don't think pulseeffects needs to depend on all its plugins. I am pretty sure you should make a PR for each package separately.

And how the user  is supposed to learn which packages to install in order to get those features working? If `xbps` had optional dependencies I could use them, but unfortunately it doesn't.

**Comment by paper42** [comment]
Screenshot from 2021-06-26 17-30-00

bottom right corner

File: srcpkgs/pulseeffects-legacy/template (hunk 1 above)
Comment (Logarithmus) [reply srcpkgs/pulseeffects-legacy/template:7]: Who would win? "Rationality & utility" vs. "convention & tradition". I'll let Void members decide :)

**Comment by Logarithmus** [comment]
@paper42 hmm OK, in this specific case of `pulseeffects` you are right.

**Comment by meator** [comment]
> > Also, "fix greyed out effects" is confusing to me. Effects which need packages that are not installed are greyed out. I don't think pulseeffects needs to depend on all its plugins. I am pretty sure you should make a PR for each package separately.
> 
> And how the user is supposed to learn which packages to install in order to get those features working? By searching the Internet and going to https://archlinux.org/packages/community/x86_64/pulseeffects/? If `xbps` had optional dependencies I could use them, but unfortunately it doesn't.

You can use [README.voidlinux](https://github.com/void-linux/void-packages/blob/master/Manual.md#documentation).

**Comment by Logarithmus** [comment]
See https://github.com/void-linux/void-packages/pull/31664, https://github.com/void-linux/void-packages/pull/31665, https://github.com/void-linux/void-packages/pull/31666, https://github.com/void-linux/void-packages/pull/31667,

**Pull request closed without merging**